content/en/docs/concepts/scheduling/configure-scheduler.md

```diff
@@ -0,0 +1,217 @@
+---
+title: How to Configure Your Kubernetes Scheduler
+reviewers:
+- wgliang
+content_template: templates/concept
+weight: 30
+toc_hide: false
+---
+
+{{% capture overview %}}
+
+This is third article of how to use and extend the Kubernetes scheduler series, the whole series of artices are,
+
+* [How does the Kubernetes scheduler work?](TODO)
+* [How to customize and extend the kubernetes scheduler?](/docs/concepts/scheduling/customization-and-extension/)
+* How to configure your kubernetes scheduler?
+* [Choose scheduling algorithms and strategies for your cluster](TODO)
+* [Learning kube-scheduler's design and implementation from source code.](TODO)
```

Comment:
Since the rest of the topics were listed in the present tense, let's follow the same approach (or start everything with "how to"? )  :)
"Learn the kube-scheduler's design and implementation from source code."